### DISCUSSION TITLE
Greeting message when use click on start conversation

### DISCUSSION THREAD

**Discussion by dubeychandan996** [chaskiq/chaskiq#299] [Ideas]
Want to send Greeting message when users click on start conversation and I want to set it from Routing Tab.
Send a message(defined from bot section) always if visitor/lead click start conversation

**Comment by michelson** [comment]
yes this is a good one! , i'm in the process of build an adaptation for that.

**Reply by dubeychandan996** [reply]
Thanks for implementing. I can see on website it looks nice. Did you push on master branch or docker. Or when it will be live.

**Reply by michelson** [reply]
yes is pushed on master & deployed in the docker image `latest` tag

**Reply by michelson** [reply]
when you upgrade you will need tun run `rails db:migrate` and restart the app (probably)

**Reply by dubeychandan996** [reply]
Thanks for the update. You are the best.

**Comment by michelson** [comment]
Hi , I forgot to mention that this is done in master branch. relased a couple of month ago as a "new conversation" bot flows
